src/components/pages/Books.js

```diff
@@ -1,24 +1,28 @@
-import { useSelector } from 'react-redux';
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable guard-for-in */
```

Comment:
- [ ] As the previous reviewer requested, please try to fix all linter errors instead of disabling them.